examples/clean-finalizer/README.md

```diff
@@ -0,0 +1,6 @@
+# Force Clean Finalizers
+
+1. Run `setup.sh` to simulate a condition of unable to clean finalizers.
+2. Press `Ctrl + C` to break
+3. Run `test.sh` to annotate chaos resource and force cleaning finalizers.
```

Comment:
I suggest demonstrating the detailed operation for force cleanup instead of executing a script. That would be much helpful for users.

Reply:
I wrote the example as an e2e test, not user oriented feature demonstration.